### DISCUSSION TITLE
Welcome to Alovoa Discussions!

### DISCUSSION THREAD

**Discussion by Nonononoki** [Alovoa/alovoa#100] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Marethyu6** [comment]
I'm Matteo, I'm kind of a FLOSS enthusiast so I liked this project even just because it's one of the few promising alternatives to a service the Internet/Web could provide (and always actually did in a way or another) otherwise in the hands of big$$ (which, given the sensitivity of data involved, is definitely unacceptable), then also because meeting people yayyy!  


That said, I fear this network has to gain people joining it in order to be effective, so here's a little feedback I hope might help:

- make settings for "search" more clear > in the sense you can access something like _advanced filters_ and, example, add more km or a global search (I always have to click on another country to see new people and there's no way around it with current filters)
- add a simple floating arrow-to-right when a profile has more than one picture > from laptop I struggled finding that I had to keep my touchpad pressed and swipe to right (and that's because I've noticed there were more pics assuming the presence of multiple dots under it, quite _common_ today, but _assuming_ is not very good for user-experience and at least a clear alternative to a _common_ gesture should be clearly shown in the interface)
- why am I not able to _rate_ some users? How does it work? I'm sure there's a reason/setting making this happen, but I can't find it, making it clear would empower users over the possibilities the tool gives I guess
- I suggest some screenshots of the interface and possible actions in the website should be featured in the homepage, as people should not be forced to create an account, in order to see what you can offer (or have I missed that content?)  

That's all for the moment, but hey, remember I believe this is a much needed and great project so just overall thanks for making it happen! ;D

**Reply by Nonononoki** [reply]
Hi, thanks for your interest :-)

1. You can currently access the advanced filter with the cog icon on top. What else do you have in mind? Also, the "global" setting is kinda there. The algorithm will search for users around the world if no fitting users are within in search area.
2. The arrow image swiping was actually in an earlier version, but it kinda looks outdated. Also, it was displaying over the actual images on smaller screens. Maybe only showing them only on big screens?
3. Because you can only rate someone if they are the "right" person for you. Currently this means that they must have the same intention, both are in each other's preferred genders and age brackets. This issue will disappear once there are more users, then no "incompatible" users will be shown at all.
4. You are right, the problem is just that everything is currently under development and visual changes often occur, and changing the screenshots that often is a chore :)

**Comment by jirafey** [comment]
I'm trying this app out, the main problem that is misleading is the ability to swipe like someone not compatible. Swiped everybody and now only guys are left. I have swiped the test account, I feel like this project needs more developers and more insight for the users. 

Also account deletion does not work, That is honestly not bad but I have made 3 accounts that are pretty much identical just to test the app. I have like everyone that is compatible. I hope messages are even possible, seems the project has had some rough days and it's kinda not active.

**Reply by Nonononoki** [reply]
Hi! This is a hobby project, I still have a full-time job. I wouldn't say that the project is not active when I don't commit in 3 days. What is the problem when deleting your account? What step in the deletion process doesn't work? I have created and deleted several test accounts myself and have not encountered any problems.

**Comment by ctlw83** [comment]
Are there any plans to do some algorithmic matching, such as based on the type of relationship, hobbies, and other keyword values?

**Comment by ip6li** [comment]
Is there a recommendation for an AI/ML platform/framework?

**Comment by Resoluciones** [comment]
Hi! Congratulations for Alovoa!

I saw today that the Android app does not show any person at "Search" section.   And regarding the online version, the contacts appear but cannot like or dislike, as the option does not appear.  Maybe it is something you are working at currently.

Another question; will it be any daily limit for "likes"?  As that maybe can be the cause of the explained situation.

Thanks!

Best regards,
Alexis

**Comment by jeffrey734** [comment]
Hi, thanks for briging this app to our world. A world completly transformed where people have to meet differently. I've passed so much time on so many dating apps over the past 7 years that I know almost each feature of the top 10 most used app. I would have so many suggestions but I see that the project isn't fully mature yet. I really hope that this will keep going on as it's too important for today's life. It's time that dating group app stop taking advantage of people feelings to make them rich. We got a point that there's even app with chatbot to make you talk with nobody then asking you to pay to continue you conversation. What a shame and totally disgusting.

I'd like to know where this app is going and what is it's status.

**Reply by Nonononoki** [reply]
A 2.0 release is being worked on, which includes big front-end changes as well and structural changes with Keycloak authentication.

Progress can be tracked here: https://github.com/Alovoa/alovoa/milestone/2

**Comment by BGebken** [comment]
Hi,
My name's Bennett and I work in Civic Tech so I have a keen appreciation for the positive or negative impact that technology can have on society. I was married for 10 years so only vaguely aware of how bad the dating apps were but now over the past few years, I have tried almost all of them and think they are all horrible. I started analyzing what was wrong with their incentives and was trying to come up with a better model when I found this project which seems to be an amazing solution!

I would love to support in any way that I can, and excited for the 2.0 release! Most of my coding expertise is data science but I have a broad horizontal skillset!

**Comment by Prepeleac** [comment]
Hi mi name is Nicolas and i want to report an delete all my accounts. What happen yesteraday i make an accont mi first accont with mi mail solonikolas@outlook.com and at night let it work always put an error has occurred try to delete and restart but nothing. Then I create another account with the same email and it accepts it. Now I have another account that also appears when I search for it.How it is possible that there are several accounts with the same email my email. Please fix this. I have already denounced my account that create first. Thank you.
